config/dev-tekton-dashboard.yaml

```diff
@@ -1,8 +1,53 @@
+# ------------------- Dashboard Service Account ------------------- #
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    k8s-app: tekton-dashboard
```

Comment:
Why `k8s-app` and not `app` like we do for the dashboard? FWIW either is fine I'd just rather us be consistent